src/main/java/com/github/bot/curiosone/core/analysis/TokenScorer.java

```diff
@@ -0,0 +1,83 @@
+package com.github.bot.curiosone.core.analysis;
+
+import static java.util.Comparator.comparing;
+
+import com.github.bot.curiosone.core.nlp.Meaning;
+import com.github.bot.curiosone.core.nlp.POS;
+import com.github.bot.curiosone.core.nlp.Token;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Finding Sentiment's score.
+ *
+ * @author Cosmo Pugliese && Francesco Natale
+ */
+
+public class TokenScorer {
+
+  /**
+   * HashSet containing tokens' score.
+   */
+  private static HashSet<Double> scoreSet = new HashSet<>();
+
+  /**
+   * Load SentiWordNet's dictionary.
+   */
+  private static Map<String, Double> dictionary = DictionaryLoader.loadDict();
+
+  /**
+   * Calculate the score of the Token.
+   *
+   * @param t is the Token add t's sentiment's score to scoreSet.
+   */
+  public static void calculateScore(Token t) {
+    if (t.getMeanings().stream()
```

Comment:
WTF!!!!!! 😧
Save the result as intermediate value and then use it for the comparison!!!